package.json

```diff
@@ -88,6 +88,7 @@
     "@babel/core": "^7.4.3",
     "@babel/preset-env": "^7.4.3",
     "acorn": "^7.2.0",
+    "assert": "^2.0.0",
```

Comment:
Since webpack 5, webpack doesn't provide automatic polyfill for Node.js built-in modules any more, so we need to these modules for browser-compatibility. And yes, webpack will resolve to `node_modules`.

Reply:
Thanks for the clarification!